dev/AppNotifications/AppNotificationDevicesData.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include "Microsoft.Windows.AppNotifications.AppNotificationDevicesData.g.h"
+
+namespace winrt {
+    using namespace Windows::Foundation::Collections;
+}
+
+namespace winrt::Microsoft::Windows::AppNotifications::implementation
+{
+    struct AppNotificationDevicesData : AppNotificationDevicesDataT<AppNotificationDevicesData>
+    {
+        AppNotificationDevicesData() = default;
+
+        // Getters
+        hstring CameraDeviceId();
+        hstring MicrophoneDeviceId();
+        hstring SpeakerDeviceId();
+
+        // Setters
+        void CameraDeviceId(hstring const& value);
+        void MicrophoneDeviceId(hstring const& value);
+        void SpeakerDeviceId(hstring const& value);
+
+        static bool IsVideoCallingSupported();
+
+    private:
+
+        hstring m_cameraDeviceId;
```

Comment:
Can have some default value either empty string or system here or in constructor

Reply:
empty String